crates/rspack_binding_values/src/module.rs

```diff
@@ -21,18 +21,41 @@ impl ToJsModule for dyn Module + '_ {
     let original_source = self
       .original_source()
       .and_then(|source| source.to_js_compat_source().ok());
+
     self
       .try_as_normal_module()
       .map(|normal_module| JsModule {
-        original_source,
-
-        resource: normal_module
-          .resource_resolved_data()
-          .resource_path
-          .to_string_lossy()
-          .to_string(),
+        original_source: original_source.clone(),
```

Comment:
use `Clone` because rustc not happy, but it is ok because once any `try_xxx` succeeds, the rest `or_else` function will not be executed.